### ISSUE TITLE
There is no -setupCoreDataStackWithiCloudContainer method with auto-migrating

### ISSUE THREAD

**Issue by Spokane-Dude** [magicalpanda/MagicalRecord#925]
Using this method:

`[MagicalRecord setupCoreDataStackWithiCloudContainer:nil localStoreNamed:@"saori.sqlite"];`

I have iCloud sync'ing with MR working (after a fashion)... the problem is that the old CoreData stack was deleted, when I expected it to be migrated.  Is there a way to easily fix this?

**Comment by davoda** [comment]
@Spokane-Dude did you happen to come across a solution? Running into the same issue.

**Comment by Spokane-Dude** [comment]
@davoda  no, I'm not sure Tony Arnold has had a chance to look at it... I looked through all of my emails from Tony and I don't see any response for this issue.  You might want to send him a note telling him that it's happening to you also.

**Comment by Alydyn** [comment]
Related to #787?

**Comment by Spokane-Dude** [comment]
no, the old CD store is deleted... has nothing to do with the periods in the name.  SD

**Comment by Spokane-Dude** [comment]
Hey Guys... what's the status of this?  I have run into this again... on a new app... unable to create a connection between CD and iCloud.

**Comment by tonyarnold** [comment]
Yeah, sorry guys — MagicalRecord has been on the back burner a lot lately. We could use help to resolve issues like this one if you have time, otherwise it'll get sorted once I eke out some time to do so.
